### ISSUE TITLE
Auto rescue command, like autobash

### ISSUE THREAD

**Issue by toasted323** [toasted323/CoffeeMud#27]
The fighter continuously rescues in combat

**Comment by ParreaBob** [comment]
Further description, literally want to mimic autobash except the command input would be rescue. I can try to recreate this via creating an ability, but I've never created an auto-ability.

**Comment by toasted323** [comment]
Is this still relevenant?

**Comment by ParreaBob** [comment]
Yes but super low priority. As we gain momentum and refine combat further the capability might still be nice to have. Loki suggested it was fairly easy to make so maybe I will poke around with it after finishing Fallback.
